pkg/agent/controller/networkpolicy/audit_logging_test.go

```diff
@@ -244,13 +253,15 @@ func TestGetNetworkPolicyInfo(t *testing.T) {
 	c := &Controller{ofClient: &openflowtest.MockClient{}}
 	ob := new(logInfo)
 	regID := openflow.APDispositionField.GetRegID()
-	dispositionMatch := openflow15.MatchField{
+	testMatch := openflow15.MatchField{
 		Class:   openflow15.OXM_CLASS_PACKET_REGS,
 		Field:   uint8(regID / 2),
 		HasMask: false,
 		Value:   &openflow15.ByteArrayField{Data: []byte{1, 1, 1, 1}},
 	}
-	matchers := []openflow15.MatchField{dispositionMatch}
+	matchers := []openflow15.MatchField{testMatch}
+	testMatch.Field = uint8(openflow.TrafficControlActionField.GetRegID() / 2)
```

Comment:
should we check disposition since the test provide an input, perhaps the test could be a table driven one to cover multiple cases.

Reply:
Updated the UT.